wagtailmarkdown/static/wagtailmarkdown/js/simplemde.attach.js

```diff
@@ -32,3 +32,17 @@ $(document).ready(function() {
         simplemdeAttach(elem.id);
     });
 });
+
+/*
+ * Used to initialize content when MarkdownFields are used in admin panels.
+ */
+$(document).on('shown.bs.tab', function(e) {
+    $('.CodeMirror').each(function(i, el){
+        setTimeout(
+            function() {
+                el.CodeMirror.refresh();
+                console.log(el);
```

Comment:
this should definitely not be in the release

Reply:
Okay then should the issue be closed?

Reply:
no, that was only meant for the `console.log()` line

Reply:
Ah I didn't see that my bad 😬

Reply:
Removed, thanks Dan